playwright.config.ts

```diff
@@ -29,6 +29,7 @@ export default defineConfig({
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
     trace: "on-first-retry",
   },
+  timeout: process.env.CI ? 60000 : 30000,
```

Comment:
So, the way I read this, if we're in CI, use a longer timeout, otherwise, use a shorter timeout?

Reply:
Yes